lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -5967,18 +5967,27 @@ namespace Js
     ScriptFunction* JavascriptLibrary::CreateScriptFunction(FunctionProxy * proxy)
     {
         ScriptFunctionType* deferredPrototypeType = proxy->EnsureDeferredPrototypeType();
+        if (proxy->GetFunctionInfo()->HasComputedName())
+        {
+            return RecyclerNewWithInfoBits(this->GetRecycler(), (Memory::ObjectInfoBits)(EnumFunctionClass | Memory::FinalizableObjectBits), ScriptFunctionWithComputedName, proxy, deferredPrototypeType);
```

Comment:
Why does this need FinalizableObjectBits now but didn't previously?

Reply:
Copy and paste error from CreateScriptFunctionWithInlineCache. 
Fixed.